cli/tools/init/mod.rs

```
@@ -238,6 +247,57 @@ Deno.test(function addTest() {
   Ok(())
 }
 
+pub async fn init_npm(name: &str, args: Vec<String>) -> Result<i32, AnyError> {
+  let script_name = format!("npm:create-{}", name);
+
+  fn print_manual_usage(script_name: &str, args: &[String]) -> i32 {
+    log::info!("{}", cformat!("You can initialize project manually by running <u>deno run {} {}</> and applying desired permissions.", script_name, args.join(" ")));
+    1
+  }
+
+  if std::io::stdin().is_terminal() {
+    log::info!(
+      cstr!("⚠️ Do you fully trust <y>{}</> package? Deno will inoke code from it will all permissions. Do you want to continue? <p(245)>[y/n]</>"),
+      script_name
+    );
+    loop {
+      let _ = std::io::stdout().write(b"> ");
```

Should we surface the error here?